src/styles/blocks/card.scss

```diff
@@ -0,0 +1,30 @@
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  max-width: 166px;
```

Comment:
More correctly set width 200px like in figma design instead of setting max-width. Also, please don't forget about `box-sizing: border-box;` rule to include padding sizes to overall card width/height